pom.xml

```diff
@@ -1937,6 +1938,14 @@
             <failIfNoTests>false</failIfNoTests>
             <excludedGroups>${test.exclude.tags}</excludedGroups>
           </configuration>
+          <executions>
```

Comment:
Without this change, `mvn -pl :java8-tests_2.11 test` wasn't triggering these tests.